File: 백준/Bronze/2869. 달팽이는 올라가고 싶다/달팽이는 올라가고 싶다.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int upLength = sc.nextInt();
        int downLength = sc.nextInt();
        int treeLength = sc.nextInt();

        int days = (treeLength - downLength) / (upLength - downLength);

        if ((treeLength - downLength) % (upLength - downLength) != 0) {
            days++;
        }
        System.out.println(days);
    }
}

File: 백준/Bronze/2869. 달팽이는 올라가고 싶다/달팽이는 올라가고 싶다.cc
#include <iostream>
using namespace std;

int main()
{
	int upLength;
	int downLength;
	int treeLength;
	int days;

	cin >> upLength >> downLength >> treeLength;

	days = (treeLength - downLength) / (upLength - downLength);

	if ((treeLength - downLength) % (upLength - downLength) != 0)
	{
		days++;
	}

	cout << days << endl;
}